services/discovery/v1.js

```diff
@@ -30,6 +31,25 @@ module.exports = function (RED) {
     endpoint = '',
     sEndpoint = 'https://gateway.watsonplatform.net/discovery/api';
 
+const ExecutionList = {
+  'createEnvrionment': executeCreateEnvrionment,
+  'listEnvrionments': executeListEnvrionments,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 2.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)